experimental/web/testing/parse_test_list.py

```diff
@@ -72,15 +72,14 @@ def parse_ctest_dump(ctest_dump_path, build_dir):
       }
 
       # Parse the 'command' list into the source file and its arguments.
-      #   /path/to/run_test.bat    # TODO(scotttodd): replace with `node`?
       #   /path/to/test_runner.js  # such as iree-check-module.js or test.js
       #   arg 1                    # such as --driver=local-task
       #   arg 2                    # such as check_vmvx_op.mlir_module.vmfb
-      test_source_absolute_path = test["command"][1]
+      test_source_absolute_path = test["command"][0]
```

Comment:
The TODO about `node` is still relevant, but I'm fine with dropping it. With a native test you can just run the test executable directly, but with JavaScript you either want to run the test under nodejs via `node script.js` or in a web browser (I'm rusty on specifics there though).

Thanks for updating this array indices here :)

Reply:
I was also looking at the android stuff and thinking about whether we should still have something like Bazel's "--run_under" that allows us to prepend every test command with some wrapper. It's just that for "normal" desktop platforms we'd have the runner be the empty string.